src/pages/home/report/comment/TextCommentFragment.tsx

```diff
@@ -48,6 +49,10 @@ function TextCommentFragment({fragment, styleAsDeleted, styleAsMuted = false, so
     const {translate} = useLocalize();
     const {isSmallScreenWidth} = useWindowDimensions();
 
+    useEffect(() => {
+        Performance.markEnd(CONST.TIMING.MESSAGE_SENT, {message: text});
```

Comment:
Well, basically this effect will be called every time (i. e. for each text message), but since there is no corresponding start event -> such measurements will be ignored. And we'll get measurements only for associated end/start events with the same text/message. I hope I answered on the question 😅